tests/pipeline/test.nf.test.snap

```diff
@@ -0,0 +1,24 @@
+{
+    "db_tables": {
+        "content": [
+            "alleles.tsv:md5,77994270b1eab3e8363c90d8d09780fa",
+            "conditions_alleles.tsv:md5,e865774e32935a69306461ff47428aa8",
+            "conditions.tsv:md5,a6fe5daaf11a90b61c83c9934969c52c",
+            "entities_proteins.tsv:md5,d008f6c7518e9932de2159751cb2519c",
+            "entities.tsv:md5,5aa057079ae45d1e0fd86ebcaa18af0f",
+            "microbiomes_entities.no_weights.tsv:md5,4d45837586c7c6dc4a19b41735215b01",
+            "microbiomes_entities.nucl.tsv:md5,f5a3ff3cc1f8100f56238b9fab405e66",
+            "microbiomes_entities.tsv:md5,37690c15b6151ff0622130b8672cddb1",
+            "microbiomes.tsv:md5,20f141ba0c97325e9d99a9c0765f9a37",
+            "proteins_peptides.tsv:md5,09f90e7895eca070c9301a4b818fa240",
+            "stats.txt:md5,83204291158696ce5d19c4e976f38efe"
+        ],
+        "timestamp": "2024-06-26T08:14:54+0000"
+    },
+    "logs": {
+        "content": [
+            "prediction_warnings.log:md5,d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
Thats the md5sum of an empty file. Here it might make sense because its a log file with warnings I assume. The md5sum makes sure that there are no warnings.

Reply:
Yes that was the intention, it shouldn't change to non empty for these tests.